examples/Question_answering_using_a_search_API.ipynb

```diff
@@ -398,7 +398,7 @@
      "text": [
       "Top 5 articles: \n",
       "\n",
-      "Title: NBA Finals: Denver Nuggets beat Miami Hea, lift thier first-ever NBA title\n",
+      "Title: NBA Finals: Denver Nuggets beat Miami Hea, lift their first-ever NBA title\n",
```

Comment:
Should change `Miami Hea` to `Miami Heat` - the report from Business Standard weirdly has 2 spelling errors in the title